src/components/r4-track.js

```diff
@@ -49,20 +65,19 @@ export default class R4Track extends LitElement {
 		return this.track ? this.renderTrack() : this.renderNoTrack()
 	}
 
-	get url() {
-		return this.origin + this.track.id
-	}
-
 	renderTrack() {
 		const t = this.track
 		return html`
+			${this.playing ? '' : ''}
```

Comment:
this is not so cool, but was required to get the component to update when the `r4-app config` changes. For sure there is a better way somewhere